packages/terra-site/src/examples/table/examples/MultiSelect.jsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+
+import Table from 'terra-table';
+
+const table = () => (
+  <div>
+    <h2>Examples</h2>
+    <h3>Multiple Selectable Rows</h3>
+    <Table isStriped={false}>
+      <Table.Header>
+        <Table.HeaderCell content={'Name'} key={'NAME'} minWidth={'small'} />
```

Comment:
This example file still has some single quotes being used. Also, this file has several examples, they should be broken out into separate files.

Reply:
Addressed the single quotes in this file. Also, I'm fine with splitting them out, I just copied this style from [MultiSelectList](https://github.com/cerner/terra-core/blob/master/packages/terra-site/src/examples/list/examples/MultiSelect.jsx). Is there a preferred way of organizing these separated files? Should there be another directory under "examples" that keeps these MultiSelectableRows example? Or should I keep them like this to be consistent with List?

Reply:
All of our other components have the examples split out, I'm not sure why that one got done differently. It should be fine to keep them all under the examples directory.

Reply:
I just assumed it was because MultiSelect and SingleSelect List had their own documentation page separate from the List component. But I went ahead and [moved all the examples for table to be directly under example](https://github.com/cerner/terra-core/pull/970/commits/213dd233bc14cb7c06b5b59958d338fcbd0b932b#diff-049c31c4f88c4154e188a37bfde6a7b4R21)